src/PHPDocsMD/DocInfoExtractor.php

```diff
@@ -171,4 +175,25 @@ private function figureOutParamDeclaration($words, $currentNameSpace)
 
         return false;
     }
+
+    /**
+     * @param $words
+     * @return array|false
+     */
+    private function figureOutSeeDeclaration($words)
+    {
+        array_shift($words);
+
+        if (!$words) {
+            return false;
```

Comment:
`$see = false;` get you the chance to use only one return statement, more readable.